core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/store/dot-experiments-reports-store.ts

```diff
@@ -228,4 +250,29 @@ export class DotExperimentsReportsStore extends ComponentStore<DotExperimentsRep
     private getLabelName(trafficProportion: TrafficProportion, variantId: string) {
         return trafficProportion.variants.find((variant) => variant.id == variantId).name;
     }
+
+    private getSummaryData(results: DotExperimentResults): DotExperimentSummary[] {
+        const summary: DotExperimentSummary[] = [];
+
+        Object.values(results.goals.primary.variants).map((variant) => {
+            //TODO: Add the traffic split when the endpoint sends the data
+
+            summary.push({
+                id: variant.variantName,
+                name: variant.variantDescription,
+                trafficSplit: 'TBD',
+                pageViews: variant.totalPageViews,
+                sessions: results.sessions.variants[variant.variantName],
+                clicks: variant.uniqueBySession.count,
+                bestVariant: variant.uniqueBySession.totalPercentage / 100,
+                improvement:
+                    (variant.uniqueBySession.totalPercentage -
+                        results.goals.primary.variants.DEFAULT.uniqueBySession.totalPercentage) /
+                    100,
+                isWinner: results.bayesianResult.suggestedWinner === variant.variantName
+            });
+        });
+
+        return summary;
+    }
```

Comment:
`.map` returns the new array, you don't need to push here.

Reply:
Moved direct to the selector, and fixed.